### DISCUSSION TITLE
Lesson 5: import "brownie" could not be resolved

### DISCUSSION THREAD

**Discussion by CryptoKnyte** [smartcontractkit/full-blockchain-solidity-course-py#52] [Q&A] [answered]
Hello all! Quite new to the whole programming space but I'm determined to learn.  Sorry if this is a basic question but I'm having problems getting the account addresses to import using brownie in vs code. 

<img width="1440" alt="Screen Shot 2021-09-18 at 11 32 07 AM" src="https://user-images.githubusercontent.com/90988255/133909489-8783f013-f9dd-4904-9f84-9a5c59f0f35f.png">

The error it shows says, Import "brownie" could not be resolved Pylance(reportMissingImports), and has yellow squiggles under the name brownie.  All the other commands do work for me though like brownie init, brownie compile, etc.

I am on a m1 mac air and have had this issue in previous lessons but restarting vscode fixed the issues when I had them before.  I've tried restarting vscode multiple times and even reset my computer but the error remains.  Lmk what I gotta do / upload for more help.  Thank you!!

**Comment by ChakibHi** [comment]
Did you try to run a script using for example "**brownie run scripts/deploy.py**"?

I always discard yellow problems from VS, and everything works fine, even if it says "Import "brownie" could not be resolved"

**Comment by AlbertoSinigaglia** [comment, marked as answer]
EDIT: TLDR: That's fine! You can ignore that. 

VSCode is not an IDE, It's more like an editor on steroids, so it's not able to do everything without configuring it correctly (manually)... there are a lot of answers on the internet about your problem, eg. [https://stackoverflow.com/questions/62878082/visual-code-on-windows-cant-find-module-after-installing-it-with-pip-using-wro](https://stackoverflow.com/questions/62878082/visual-code-on-windows-cant-find-module-after-installing-it-with-pip-using-wro
)

**Reply by PatrickAlphaC** [reply]
@AlbertoSinigaglia I edited a little, thanks as always for your answers!

**Reply by CryptoKnyte** [reply]
Thank you guys very much!  I'm still looking into getting everything configured correctly but it all runs regardless of the yellow squiggles.  Back to the tutorial for me :)
